modules/convert/user.go

```diff
@@ -48,6 +48,11 @@ func toUser(user *models.User, signed, authed bool) *api.User {
 		Location:    user.Location,
 		Website:     user.Website,
 		Description: user.Description,
+		// counter's
+		Followers:    user.NumFollowers,
+		Following:    user.NumFollowing,
+		StarredRepos: user.NumStars,
+		PublicRepos:  int(models.CountUserRepositories(user.ID, false)),
```

Comment:
convert.ToUser is called A LOT - this seems unwise.

Reply:
so drop `PublicRepos` ? @zeripath

Reply:
I guess so or it needs to be cached somehow.

The problem is that user convert is called a lot. So that count is going to be called all of the time as it currently stands.